src/Illuminate/Console/Scheduling/CacheOverlappingStrategy.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Illuminate\Console\Scheduling;
+
+use Illuminate\Contracts\Cache\Repository as Cache;
+
+class CacheOverlappingStrategy implements OverlappingStrategy
+{
+    /**
+     * @var \Illuminate\Contracts\Cache\Repository
+     */
+    private $cache;
+
+    public function __construct(Cache $cache)
+    {
+        $this->cache = $cache;
+    }
+
+    public function prevent(Event $event)
+    {
+        $this->cache->put($event->mutexName(), true, 1440);
```

Comment:
Would be nice to be able to modify this number in configs.

1440 = 24 hours.

Depending on your workload, if one of your servers goes down, you might not want to wait a full day for the key to expire and resume schedule.